contracts/Shortfall/Shortfall.sol

```diff
@@ -287,6 +290,9 @@ contract Shortfall is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable, IShor
      * @custom:access Restricted to owner
      */
     function startAuction(address comptroller) public onlyOwner {
+        PoolRegistryInterface.VenusPool memory pool = PoolRegistry(poolRegistry).getPoolByComptroller(comptroller);
+        require(pool.comptroller == comptroller, "comptroller doesn't exist pool registry");
```

Comment:
Here we are getting pool through comptroller and then verifing the pool's comptroller address to comptroller address. Is this require necessary here?

Reply:
Same comment as before